Sources/StreamChatUI/ChatChannel/ChatChannelVC.swift

```diff
@@ -67,9 +67,16 @@ open class ChatChannelVC: _ViewController,
         messageListVC.listView.isLastCellFullyVisible
     }
 
+    internal var isViewVisible: ((ChatChannelVC) -> Bool) = { channelVC in
```

Comment:
nit: Why not a function? If it is only for tests, you can make it a function and add the vc.view on a dummy window in your tests

Reply:
It makes the tests more simple and also a tiny bit faster most likely. IMO it is not worth to change it now.

Also, it can't be nil, otherwise we can't change it in the test 👍